javascript/javascript2/week3/fastestPresserredo.js

```diff
@@ -0,0 +1,82 @@
+const timeInput = document.querySelector('#timeInputMarkup');
+const startGameButton = document.querySelector('#startGameMarkup')
```

Comment:
There is a very subtle difference between queryselector and getelementbyid. Since you can have many elements with the same class in HTML, we use the query selector when we want all of the elements in that class. Ids in HTML have to be unique, so you use the getelementbyid in cases when you just need one to retrieve one element from HTML, which is true in this case! So although they have the same functionality here, it's a good practice to have to get the element by id instead of query selector.